File-Formats/DocIO/Working-with-Charts.md

```diff
@@ -759,6 +759,82 @@ document.Close()
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Charts/Applying-3D-formats).
 
+## Removing Chart Title
```

Comment:
We have separate page for Chart Title, 
As we mentioned in the task goals, please add as last topic in this page https://help.syncfusion.com/file-formats/docio/charts/chart-title

Reply:
Addressed the feedback